modules/post/multi/manage/shell_to_meterpreter.rb

```diff
@@ -43,6 +43,10 @@ def initialize(info = {})
                   [true, 'Which method to try first to transfer files on a Windows target.', 'POWERSHELL', ['POWERSHELL', 'VBS']]),
       OptString.new('PAYLOAD_OVERRIDE',
                     [false, 'Define the payload to use (meterpreter/reverse_tcp by default) .', nil]),
+      OptString.new('PLATFORM_OVERRIDE',
```

Comment:
Was there a reason for not using the previously recommended OptEnum approach in the end 👀 

https://github.com/rapid7/metasploit-framework/blob/d8f36d65a8b6a24ef018966334a248728d082e9a/lib/msf/core/auxiliary/mms.rb#L33